src/Modal.js

```diff
@@ -166,59 +198,70 @@ class Modal extends React.Component {
 
   renderChildren() {
     const {
-      className,
       wrapClassName,
       modalClassName,
       backdropClassName,
-      contentClassName,
       cssModule,
       isOpen,
-      size,
-      backdrop,
-      children,
-      ...attributes
-    } = omit(this.props, ['toggle', 'keyboard', 'onEnter', 'onExit', 'zIndex']);
+      backdrop
+    } = this.props;
+
+    const modalAttributes = {
+      onClickCapture: this.handleBackdropClick,
+      onKeyUp: this.handleEscape,
+      style: { display: 'block' },
+      tabIndex: '-1'
+    };
+
+    if (this.hasTransition()) {
+      const modalTransitionTimeout = 300;
+      const backdropTransitionTimeout = 150;
```

Comment:
Should these times be configurable, or should that be another PR. At a certain point we could make the transition timing variable and when it is 0 disabled the transition animation.